src/pages/docs/continuous-integration/bitrise-ci.md

```
@@ -8,98 +8,104 @@ contextual_links:
 
 ---
 
-Bitrise is a CI tool specifically designed for mobile continuous integration and delivery system for iOS and Android native apps. Bitrise is an enterprise product and hence provides support for build executions on Linux and Mac Os as well.
- 
+Bitrise is a CI tool specifically designed for mobile continuous integration and delivery system for iOS and Android native apps. Bitrise is an enterprise product and hence provides support for build executions on Linux and macOS as well.
+
 ---
 
 > <p id="prerequisites">Prerequisites</p>
 > 
 >    - User needs a repository where the steps mentioned in a shell script file can be uploaded and updated. 
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)